src/RadixSortLSD.chpl

```diff
@@ -85,6 +98,12 @@ module RadixSortLSD
       }
     }
 
+    inline proc getDigit(key: _tuple, rshift: int, last: bool, negs: bool): int 
+        where isHomogeneousTuple(key) && key.type == key.size*uint(bitsPerDigit) {
+      return key[key.size - rshift/bitsPerDigit];
```

Comment:
This assumes 1-up indexing, right? How could we make this agnostic to indexing?

Reply:
Yeah, this and 2 other places (`getBitWidth`, and `var curDigit = 1 + ...`) assume 1-up and will have to be adjusted.

Reply:
@bradcray made changes like these in some other PRs like PR #351 and #352. Do we need to change this for 1.21 and 1.22 compatibility?

Reply:
Yeah, we'll need to do something to make it compatible with 1.22. I was leaning towards something like https://github.com/mhmerrill/arkouda/pull/352. 

I'll rebase and make that change tomorrow.

Reply:
Rebased and made 1.20-1.22 compatible. I regathered performance numbers with the new coargsort.py benchmark and included them in the description.